swan.sh

```diff
@@ -1,44 +0,0 @@
-package: SWAN
```

Comment:
Yes that's the idea. I would not add extra packages, I would prefer to rebuild the generators. In any case tests are being performed on the nightlies part because we don't want to pollute the production environment.